sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -3681,6 +3681,48 @@ object GpuOverrides extends Logging {
         TypeSig.ARRAY + TypeSig.DECIMAL_64).nested(), TypeSig.all),
       (sample, conf, p, r) => new GpuSampleExecMeta(sample, conf, p, r)
     ),
+    exec[SubqueryBroadcastExec](
+      "Plan to collect and transform the broadcast key values",
+      ExecChecks(TypeSig.all, TypeSig.all),
+      (s, conf, p, r) => new SparkPlanMeta[SubqueryBroadcastExec](s, conf, p, r) {
+
+        override val childExprs: Seq[BaseExprMeta[_]] = Nil
+
+        override val childPlans: Seq[SparkPlanMeta[SparkPlan]] = Nil
+
+        override def tagPlanForGpu(): Unit = s.child match {
+          case ex @ BroadcastExchangeExec(_, c2r: GpuColumnarToRowExecParent) =>
```

Comment:
I am more than a little confused. When exactly does this happen? Moving the comments from below up closer to the top would be good.

Reply:
Done.